src/unit/categories.js

```diff
@@ -0,0 +1,11 @@
+define(function() {
+
+/**
+ * categories()
+ *
+ * Return all unit categories.
+ */
+return [ "acceleration", "angle", "area", "digital", "duration", "length", "mass", "power",
```

Comment:
Yes, it helps to make the build easier. This could change though when the source code uses ES6.